tools/enzyme/README.md

```diff
@@ -5,3 +5,23 @@
 Enzyme needs to be compiled against a specific version of LLVM. You are strongly advised to use the Dockerfile for this one.
 
 [Enzyme]: https://enzyme.mit.edu/
+
+## Commentary
+
+### Multithreading
+
+The approach here is very similar to the one for [manual](../manual),
```

Comment:
This sort of relative link is error-prone; please replace it with one using a leading slash.